spring-cloud-dataflow-composed-task-runner/pom.xml

```diff
@@ -24,6 +24,7 @@
 	<packaging>jar</packaging>
 	<properties>
 		<plexus.utils.version>3.3.0</plexus.utils.version>
+		<java-cfenv.version>2.2.0.RELEASE</java-cfenv.version>
```

Comment:
No version definition here, it's already in parent pom.

Reply:
This needs to be removed. I will remove while merging the PR.